utilities/workers/worker_manager.py

```diff
@@ -106,6 +111,18 @@ def get_worker_target(self) -> "(...) -> object":  # type: ignore
         """
         return self.__target
 
+    def get_input_queues(self) -> "list[queue_proxy_wrapper.QueueProxyWrapper]":
+        """
+        Returns the input queues.
+        """
+        return self.__input_queues
+
+    def get_worker_name(self) -> str:
+        """
+        Returns the name of the worker type.
+        """
+        return self.__worker_name
```

Comment:
Is it possible to get the name of the target function instead?

Reply:
We would have to create getter in each worker I think. Would this be a good idea, I'm not sure?

Reply:
I tried this and it worked:
```py
def test():
    pass


if __name__ == "__main__":
    a = test
    print(a.__name__)
```
So I think you could have:
```py
def get_target_name(self) -> str:
    return self.__target.__name__
```

Reply:
Actually, you can directly use the worker's process name instead:
```py
for worker in self.__workers:
    print(worker.name)
```
Source: https://docs.python.org/3/library/multiprocessing.html#multiprocessing.Process.name

Reply:
using worker.name would print out "Worker died, restarting Process-5". I'm not sure how useful this would be, should we be concatenating worker.name with the target name?

Reply:
That would work.